templates/tensorflow-gpu-1.12/image_classifier/src/example_test.py

```diff
@@ -0,0 +1,22 @@
+from .example import apply, get_model
+import Algorithmia
+
+"""
+This is the testing file, which should be bundled along with the Algorithm.py file when creating a template.
+These tests are run during the creation of a template, and if they fail - the templating process fails as well.
```

Comment:
we won't be running any tests while creating templates for the current iteration - i'm not sure if this file should be removed or not

Reply:
I kinda want to have a x_test.py file there that actually does something, but I can remove it / remove the comment

Reply:
I'll remove it for now, KISS.

Reply:
@zeryx wanted to put that test in as a best practice that we are promoting. Also our existing Hello app doeos have a test too, so it would be consistent. I am fine with either.

Reply:
Jai's point was my understanding too.

Unless we generate the standard `algo_test.py` somewhere else, I don't see why it shouldn't be included here either.

Reply:
up to you guys, I'm in different - what do you think @pmcq ?

Reply:
i felt that the comment was misleading since these tests aren't run during creation of a template or anything.  as far as the file goes, i don't really have a preference but should maybe follow our existing template https://github.com/algorithmiaio/langpacks/tree/master/python/template/src (i.e. called `__ALGO___test.py` (__ALGO__ will be substituted automatically)